src/messages/components/ConversationAddReactionInner.vue

```diff
@@ -1,8 +1,11 @@
 <template>
   <div
     class="row"
-    style="margin: 2px 4px"
+    style="margin: 2px 4px; width: 185px"
```

Comment:
As we are trying to minimize inline styles: can you move this to the style section?

Reply:
Thanks, I moved it. Changed some numbers to allow 5 emoji in each row.